resources/web/wwi/protoVisualizer/Parameter.js

```diff
@@ -211,7 +210,6 @@ export default class Parameter {
         const action = {};
         action['id'] = this.node.id;
         action[this.name] = this.value.toJson();
-        console.log('setPose', action);
         view.x3dScene.applyPose(action);
```

Comment:
I think we should finally rename the apply pose function as we use it for anything now, something like `applyAction` perhaps?